### ISSUE TITLE
Can't compile: PPCoordinatorDelegate, PPOverlayViewController, PPRecognizerResult, PPUsdlRecognizerSettings... undeclared after migrating from 2.9.0 to 4.11.0

### ISSUE THREAD

**Issue by denismullaraj** [BlinkID/blinkid-ios#255]
#208 
# [REQUIRED] Step 2: Describe your environment

  * Xcode version: 10.2.1
  * BlinkID SDK version: 4.11.0
  * What are you trying to scan? <Not compiling yet>

### [REQUIRED] Step 3: Describe the problem
I actually updated the pod version from 2.9.0 of the library to 4.11.0 and tried to compile but has issues with PPCoordinatorDelegate, PPOverlayViewController, PPRecognizerResult, PPUsdlRecognizerSettings.. and others saying "Use of undeclared...".

#### Relevant Code:

```
import MicroBlink

class MyViewController: UIViewController,  PPCoordinatorDelegate, PPScanningDelegate {

  override viewDidLoad() {
     super.viewDidLoad()
     let overlayViewController = myStoryboard.instantiateViewController(withIdentifier: "CameraViewController") as! PPOverlayViewController
     ....
  }

  func scanningViewController(_ scanningViewController: (UIViewController & MBRecognizerRunnerViewController)?, didOutputResults results: [PPRecognizerResult]) {
    ...
  }

}

```

**Comment by juraskrlec** [comment]
Hi @denismullaraj 

BlinkID v4.x is all new API, and you can read about it [here](https://microblink.com/blog/major-change-of-the-api-and-in-the-license-key-formats) and you can read our README in this repository. There are examples in repo of how to integrate v4.x with new API.

**Comment by denismullaraj** [comment]
I don't see my specific issue in the guide. I had some things that I needed to change like PPScanningViewController to MBRecognizerRunnerViewController and PPViewControllerFactory to MBViewControllerFactory. But apart from this renaming, I can't see why building the project now tells me that those protocols/classes/interfaces are "undeclared". I just bumped the version through cocoapods to 4.x (tried both with 4.11.0 and 4.0.0 and still same issue).
